.gitignore

```diff
@@ -1,6 +1,5 @@
 .idea
 node_modules
 *.log
-lib
```

Comment:
Why are we checking in `lib` now?

Reply:
Because `lib` is what's shipped as part of the package on NPM. Since we're now importing it directly from Github we need to have these built files available to us.

Reply:
We could publish this package to `@khanacademy/react-loadable`.  Maybe we could do that in the future.  I'd like for us to avoid checking in built files if possible.

Reply:
> I had to add the built files to the repo so that we can import this module using the commit SHA.

I should spend more time reading PR summaries.